src/pages/Mission/Mission.js

```diff
@@ -0,0 +1,42 @@
+import { useDispatch } from 'react-redux';
+import { joinMission, leaveMission } from '../../redux/Missions/Missions';
+
+const Mission = (mission) => {
+  const dispatch = useDispatch();
+  const {
+    id,
+    name,
+    description,
+    reserved,
+  } = mission;
```

Comment:
- [ ] Since the mission data is globally available through the redux store, It would be better not to pass props here. instead, you should use `useSelector` hook. The code will look something like `const rockets = useSelector(state => state.missions);`
- [ ] The above solution should clear the prop errors displayed in the browser console.